scipy/stats/stats.py

```diff
@@ -5173,6 +5174,113 @@ def friedmanchisquare(*args):
 
     return FriedmanchisquareResult(chisq, distributions.chi2.sf(chisq, k - 1))
 
+BrunnerMunzelResult = namedtuple('BrunnerMunzelResult', ('statistic', 'pvalue'))
+
+
+def brunnermunzel(x, y, alternative="two-sided", distribution="t"):
+    """
+    Computes the Brunner-Munzel test on samples x and y
+
```

Comment:
this needs to specify the null hypothesis, here or more detailed in the Notes section